sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/ActivityExtensions.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using OpenTelemetry.Exporter.AzureMonitor.Models;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Exporter.AzureMonitor
+{
+    internal static class ActivityExtensions
+    {
+        private static readonly string INVALID_SPAN_ID = default(ActivitySpanId).ToHexString();
+        private static readonly string INVALID_TRACE_ID = default(ActivityTraceId).ToHexString();
+
+        internal static string GetSpanId(this ActivitySpanId activitySpanId)
```

Comment:
Why not take `this Activity activity` as parameter, to do full justice to the class name.

Reply:
good point, will get his fixed.